test/runtests.jl

```diff
@@ -173,6 +173,36 @@ end
         Aqua.test_all(SlackThreads; ambiguities=false)
     end
 
+    @testset "Utilities" begin
+        format_message_counts = (i, n) -> ""
+        messages = SlackThreads.combine_texts(["abcdef", "abc"]; max_length=1,
+                                              format_message_counts)
+        @test messages == ["abcdef", "abc"] # 2 messages
+
+        messages = SlackThreads.combine_texts(["abcdef", "abc", "d"]; max_length=4,
```

Comment:
Would be good to add a test where multiple subsequent strings exceed `max_length`